bucket/docker-machine.json

```
@@ -0,0 +1,26 @@
+{
+    "version": "0.16.2",
+    "description": "Docker Machine lets you create Docker hosts on your computer, on cloud providers, and inside your own data center. It creates servers, installs Docker on them, then configures the Docker client to talk to them.",
+    "homepage": "https://github.com/docker/machine",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/docker/machine/releases/download/v0.16.2/docker-machine-Windows-x86_64.exe#/docker-machine.exe",
+            "hash": "dcf774857069749b5d973f85b15d926544a7c5556c3478c45b4f35cf86494130"
+        }
+    },
+    "bin": "docker-machine.exe",
+    "checkver": {
+        "github": "https://github.com/docker/machine"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/docker/compose/releases/download/v$version/docker-machine-Windows-x86_64.exe#/docker-machine.exe"
+            }
+        },
+        "hash": {
+            "url": "https://github.com/docker/machine/releases/download/v$version/sha256sum.txt"
+        }
+    }
```

```suggestion
    "bin": "docker-machine.exe"
```

The repo is archived, so remove `checkver` and `autoupdate`.